File: src/sdk-reference/csharp/1/error-handling/snippets/error-handling.cs
try {
  kuzzle.index.create("nyc-open-data");
  // Second call fail
  kuzzle.index.create("nyc-open-data");
} catch (BadRequestException e) {
  Console.WriteLine("Status: " + e.status() + " Message: " + e.getMessage());
  Console.WriteLine("Try with another name!");
}

File: src/sdk-reference/cpp/1/error-handling/snippets/error-handling.cpp
try {
  kuzzle->index->create("nyc-open-data");
} catch (kuzzleio::BadRequestException& e) {
  std::cout << "Status: " << e.status << " Message: " << e.what() << std::endl;
  std::cout << "Try with another name!" << std::endl;
}
